assignments/objects.js

```diff
@@ -18,31 +18,65 @@ const example = {
 }
 
 // Write your intern objects here:
+const mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+const kennan = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  "speak":  "Hello, myname is Kennan!"
+}
+const keven = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+const gannie = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+const antonietta = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
```

Comment:
```
	const antonietta = {
  "id": 5,
  "name": "Antonietta",
  "email": "adaine5@samsung.com",
  "gender": "F",
  multiplyNums: function(a,b) {
    return a*b;
  }
}

console.log(antonietta.multiplyNums(3,4))

	const antonietta = {
  "id": 5,
  "name": "Antonietta",
  "email": "adaine5@samsung.com",
  "gender": "F",
  multiplyNums(a,b){
    return a*b
  }
}

console.log(antonietta.multiplyNums(3,4))

	const antonietta = {
  "id": 5,
  "name": "Antonietta",
  "email": "adaine5@samsung.com",
  "gender": "F",
  multiplyNums : (a,b) => a*b
}

console.log(antonietta.multiplyNums(3,4))
```

Reply:
I think you have some syntax issues here.  You're missing a comma after the gender property.  Also writing methods for objects, you can't use arrow functions with the `this` keyword as there is no `this` bound to arrow functions. Also, `this.a` refers to the `a` property within the antonietta object, while a and b should be parameters.